unittests/CppInterOp/VariableReflectionTest.cpp

```diff
@@ -93,7 +93,7 @@ TEST(VariableReflectionTest, GetVariableType) {
 
 #define CODE                                                    \
   int a;                                                        \
-  const int N = 5;                                              \
```

Comment:
We can’t change that because this changes the intent of the test. Maybe use it in the struct definition below to initialize some member.